packages/terra-switch/translations/pt.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Terra.switch.switchstatuslabel.on": "ATIVAR",
```

Comment:
I imagine we have the same capitalization concerns for these translations as well.

Reply:
updated translations [here](https://github.com/cerner/terra-core/pull/2846/commits/5040f877bc1adf7e1e1b8b7f1dc296170f94c75f)